### PR TITLE
Update pagination styles

### PR DESCRIPTION
**Pull request by andresriveratoro**
## Description
Part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/35291

## Acceptance criteria
- [x] Update Pagination styles 

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by andresriveratoro** [comment]
The diff here makes it difficult to read because I moved the media queries to the bottom and updated/simplified them so they don't override other styles.

**Review by pdavies88 — APPROVED** [review]

File: packages/formation/sass/modules/_va-pagination.scss [hunk 1]
```diff
@@ -12,54 +11,122 @@
...
+    border: 0;
+    color: $color-link-default;
+    display: inline-flex;
+    font-size: 0.9em;
+    font-weight: 400;
+    justify-content: center;
+    line-height: 2;
+    padding: 0;
+    transition: all 0.3s ease-in-out;
+    transition-property: color, background-color;
```
Comment (bkjohnson) [inline packages/formation/sass/modules/_va-pagination.scss:27]: Are all of these styling changes based on a design? I'm not sure if they're representing uswds styling, something in app code, etc.

File: packages/formation/sass/modules/_va-pagination.scss [hunk 2]
```diff
@@ -12,54 +11,122 @@
...
+  }
+
+  // va-pagination-prev and va-pagination-next
+  &-prev,
+  &-next {
```
Comment (bkjohnson) [inline packages/formation/sass/modules/_va-pagination.scss:32]: I really like that you're writing the full name out - there have been several occasions where I've had a tricky time searching for something due to these sass shortcuts.

**Comment by andresriveratoro** [comment]
This is no longer required because the work for https://github.com/department-of-veterans-affairs/component-library/pull/264 was limited to updating anchors.

**Pull request closed without merging**